commerce_coordinator/apps/order_fulfillment/serializers.py

```diff
@@ -0,0 +1,18 @@
+"""Serializers for LMS (edx-platform) service"""
+from rest_framework import serializers
+
+from commerce_coordinator.apps.core.serializers import CoordinatorSerializer
+
+
+class FulfillOrderWebhookSerializer(CoordinatorSerializer):
+    """
+    Serializer for Fulfill Order Webhook input validation.
+    """
+    fulfillment_type = serializers.CharField()
+    is_fulfilled = serializers.BooleanField()
+    entitlement_uuid = serializers.CharField(required=False, allow_null=True)
+    order_id = serializers.CharField()
+    order_version = serializers.CharField()
+    line_item_id = serializers.CharField()
```

Comment:
This should be a required field (along with the others necessary) as we cannot perform operations if there are null.

Reply:
() means its required unless we put `allow_null=explicitly`